pkg/kubelet/hyper/hyperclient.go

```diff
@@ -527,20 +526,22 @@ func (client *HyperClient) PullImage(image string, credential string) error {
 	return nil
 }
 
-func (client *HyperClient) CreatePod(podArgs string) (map[string]interface{}, error) {
-	glog.V(5).Infof("Hyper: starting to create pod %s", podArgs)
-	body, _, err := client.call("POST", "/pod/create", podArgs, nil)
+func (client *HyperClient) CreatePod(podArgs string) (string, error) {
```

Comment:
`podArgs` should be changed to `grpctypes.UserPod` so as to avoid json marshaling.

Caller inside Kubelet should also be updated togather with this.